src/test/java/org/openstreetmap/atlas/checks/validation/tag/StreetNameCheckTest.java

```diff
@@ -0,0 +1,191 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Tests for {@link StreetNameCheck}
+ *
+ * @author v-naydinyan
+ */
+public class StreetNameCheckTest
+{
+    @Rule
+    public StreetNameCheckTestRule setup = new StreetNameCheckTestRule();
+
+    @Rule
+    public ConsumerBasedExpectedCheckVerifier verifier = new ConsumerBasedExpectedCheckVerifier();
+
+    private final StreetNameCheck check = new StreetNameCheck(
```

Comment:
Love the detailed unit tests. Could you also add a unit test to test the config values using "ConfigurationResolver.inlineConfiguration" ?

Reply:
Thank you for looking over this. Would you like me to add just a couple of them, or try to get most possible scenarios?

Reply:
I think just one would do.

Reply:
All done! Thank you